dev/12.系统设置/01.账号系统集成/01.oauth2.md

```diff
@@ -23,5 +23,19 @@ Zadig 支持用户通过 OAuth 2.0 账号授权进行登录。 点击`系统设
 | 用户邮箱属性             | 用户邮箱属性                                          |
 | Scopes                 | 向账号系统请求的相关字段，可查阅账号系统相关文档            |
 
+> 如果您的 OAuth provider 返回的 userinfo 信息不属于 root 层级，则需要通过 `.` 的方式嵌套配置。
+>
+> 例如在以下 userinfo 返回的样例中，如需正确解析 userinfo 返回的数据，则需将用户名属性设置为 `data.user_id`，将用户昵称属性设置为 `data.fullname`， 将用户邮箱属性设置为 `data.email`。
+```
+{
+    "code": 0,
+    "data": {
+        "user_id": "zzz",
+        "fullname": "yyy",
+        "email": "xxx"
+    }
```

Comment:
to reach better readability and understanding, we suggest:

1. more precision struct to show the demo
2. set relative value for `user_id`/`fullname`/`email`

eg, we can set the struct as below, then use `data.userinfo.user_id`、`data.userinfo.fullname`、`data.userinfo.email` to get the value we want.

```
{
    "code": 0,
    "data": {
        "userinfo": {
            "user_id": "2022001",
            "fullname": "demo",
            "email": "demo@example.com"
        },
        "timestamp": 1655882773,
        ...
    }
    ...
}
```

Reply:
updated